docs/content/docs/libs/state_processor_api.md

```diff
@@ -26,7 +26,7 @@ under the License.
 
 # State Processor API
 
-Apache Flink's State Processor API provides powerful functionality to reading, writing, and modifying savepoints and checkpoints using Flink’s DataStream API under `BATCH` execution.
+Apache Flink's State Processor API provides powerful functionality to reading, writing, and modifying savepoints and checkpoints using Flink’s [DataStream API]({{< ref "#datastream-api" >}}) and [Table API]({{< ref "#table-api" >}}) under `BATCH` execution.
```

Comment:
nit: powerful functionality to -> powerful functionality for

Reply:
This is not added by me but let's make it better so changed.